unity-renderer/Assets/DCLPlugins/RealmsPlugin/RealmsBlockerModifier.cs

```diff
@@ -0,0 +1,20 @@
+using DCL;
+using Decentraland.Bff;
+
+namespace DCLPlugins.RealmsPlugin
+{
+    public class RealmsBlockerModifier : IRealmsModifier
+    {
+        
+        public void OnEnteredRealm(bool isCatalyst, AboutResponse realmConfiguration) =>
+            Environment.i.world.blockersController.SetEnabled(isCatalyst || ShouldGreenBlockersBeActive(realmConfiguration.Configurations.CityLoaderContentServer));    
```

Comment:
could you inject the world instance? instead of referring to `Environment.i.world`